sdk/core/Azure.Core.TestFramework/src/IgnoreOnNet5Attribute.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using NUnit.Framework;
+using NUnit.Framework.Interfaces;
+using NUnit.Framework.Internal;
+
+namespace Azure.Core.TestFramework
+{
+    /// <summary>
+    /// Marks that a test should not be executed in NET5 test runs.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Method | AttributeTargets.Class | AttributeTargets.Assembly, AllowMultiple = false, Inherited = false)]
+    public class IgnoreOnNet5Attribute : NUnitAttribute, IApplyToTest
+    {
+        private readonly string _reason;
+
+        public IgnoreOnNet5Attribute(string reason)
+        {
+            _reason = reason;
```

Comment:
Assert this isn't null or empty, and maybe just require a link to actually fix the incompatibility.